### ISSUE TITLE
Does the checkpoint at stage 2 init lose autoregressive capabilities from stage 1 post?

### ISSUE THREAD

**Issue by ChintanTrivedi** [PKU-YuanGroup/Helios#134]
I was able to see the model develop autoregressive capabilities in validation samples from finishing stage 1 init and stage 1 post training (using merge lora for wan in the middle). 

Then, I did merge lora for helios using stage 1 post output, and load in stage 2 init training. The first validation sample of this traninig (step 1) is meaningless colors, but the model does start learning meaningful content again quickly. However, the validation samples in subsequent steps of stage 2 init do not seem to have autoregressive property, ie, each chunk is distinct video with frame/scene jumps in between. Is this expected behavior at this stage of training?

**Comment by SHYuanBest** [comment]
Thanks for your interest! At the start of Stage 2 init, meaningless colors will appear, but I recall it still retains the autoregressive property.

**Comment by ChintanTrivedi** [comment]
Thank you for the great work and for your response. It seems that the model recovers AR capability after training more, but it seems to be randomly gained or lost as training progresses (also true for stage 2 init as well as post). I am currently stopping training and restarting from last good checkpoint if I notice it collapsing, so this issue can be closed.

**Comment by SHYuanBest** [comment]
> Thank you for the great work and for your response. It seems that the model recovers AR capability after training more, but it seems to be randomly gained or lost as training progresses (also true for stage 2 init as well as post). I am currently stopping training and restarting from last good checkpoint if I notice it collapsing, so this issue can be closed.

You can try lowering the learning rate or increasing the batch size.
